tests/lib/rules/use-brace-expansion.js

```diff
@@ -16,6 +16,7 @@ eslintTester.run('use-brace-expansion', rule, {
     { code: '{ test: computed(function() {}) }' },
     { code: '{ test: computed("a.test", "b.test", function() {}) }' },
     { code: '{ test: computed("a.{test,test2}", "b", function() {}) }' },
+    { code: '{ test: computed("model.a.{test,test2}", "model.b.{test3,test4}", function() {}) }' },
```

Comment:
Good catch -- Adding `{ code: '{ test: computed("foo.bar.{name,place}", "foo.qux.[]", "foo.baz.{thing,@each.stuff}", function() {}) }' },` to `valid` caused a test failure, so my implementation was incomplete.